test/routes-test/claim.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+var mongoose   = require('mongoose');
+var ObjectId   = mongoose.Types.ObjectId;
+
+var should = require('should');
+var config = require('../config');
+var app = require(config.projectRoot + '/app');
+var seedData = require('../../seed_data/seedData');
+var seedDb = require('../../seed_data/seedDb');
+var utils = require('../../seed_data/utils');
+var request = require('supertest');
+
+describe('Claim-post test: ', function() {
+
+  describe('POST /claim', function() {
+
+    before(seed);
+    after(utils.dropDb);
```

Comment:
change this to also close the connection (see the edit aron did on `develop` on all the other route tests)